services/notifications/pkg/channels/channels.go

```diff
@@ -114,7 +114,10 @@ func (m Mail) SendMessage(ctx context.Context, message *Message) error {
 
 	email := mail.NewMSG()
 	if message.Sender != "" {
-		email.SetFrom(fmt.Sprintf("%s via %s", message.Sender, m.conf.Notifications.SMTP.Sender)).AddTo(message.Recipient...)
+		tmpSender := fmt.Sprintf("%s via %s", message.Sender, m.conf.Notifications.SMTP.Sender)
+		tmpParts := strings.SplitN(tmpSender, "<", 2)
+		tmpSender = fmt.Sprintf("\"%s\" <%s", tmpParts[0], tmpParts[1])
```

Comment:
Also note https://www.rfc-editor.org/rfc/rfc5322#appendix-A.1.2

If the display name itself contains double-quotes, then those need to be escaped with `\`
`"Giant; \"Big\" Box" <sysservices@example.net>`

But maybe you first can make the fix that is here, then do something about display names that have double-quotes in them.

Reply:
We could try to use a parser https://pkg.go.dev/net/mail#ParseAddress ?

Reply:
The problem with the parses is, that it thinks that a comma also separates two mails. So we would get the same error if the Mail is not already in RFC 5322 standard. The only solution would be to add the Address type. But then it would need the exact code I wrote as well to add the values and it would still need the code to accommodate the double-quotes issue.